pkg/clipboard/clipboard_test.go

```diff
@@ -3,21 +3,34 @@ package clipboard
 import (
 	"bytes"
 	"context"
+	"github.com/atotto/clipboard"
```

Comment:
The move here is unnecessary and will make [gci](https://github.com/daixiang0/gci) unhappy IMO.

Reply:
Fixed in 7ffa1037